ui/apps/platform/src/Containers/Violations/Details/ViolationDetailsPage.tsx

```diff
@@ -66,17 +66,24 @@ function ViolationDetailsPage(): ReactElement {
         return <ViolationNotFoundPage />;
     }
 
-    const { policy, deployment, resource, commonEntityInfo, enforcement } = alert;
+    const { policy, enforcement } = alert;
     const title = policy.name || 'Unknown violation';
-    const { name: entityName } = resource || deployment || {};
-    const resourceType = resource?.resourceType || commonEntityInfo?.resourceType || 'deployment';
+    /* eslint-disable no-nested-ternary */
+    const entityName =
+        'resource' in alert
+            ? alert.resource.name
+            : 'deployment' in alert
+            ? alert.deployment.name
+            : '';
+    /* eslint-enable no-nested-ternary */
```

Comment:
Is there a benefit to ternary's here over just if/else so that you don't have to disable the linter?

```javascript
let entityName = '';

if ('resource' in alert) {
    entityName = alert.resource.name;
} else if ('deployment' in alert) {
    entityName = alert.deployment.name;
}
```